src/Microsoft.AspNetCore.Sockets.Abstractions/ConnectionList.cs

```diff
@@ -34,8 +33,7 @@ public void Add(ConnectionContext connection)
 
         public void Remove(ConnectionContext connection)
         {
-            ConnectionContext dummy;
-            _connections.TryRemove(connection.ConnectionId, out dummy);
+            _connections.TryRemove(connection.ConnectionId, out var dummy);
```

Comment:
Given that @davidfowl is in Hawaii, I think the only way this nit is being addressed is if you change it yourself :P